contracts/src/v0.8/shared/test/call/CallWithExactGas.t.sol

```diff
@@ -149,6 +152,54 @@ contract CallWithExactGas__callWithExactGas is CallWithExactGasSetup {
   }
 }
 
+contract CallWithExactGas_callWithExactGasAndConsumeAllGas is CallWithExactGasSetup {
+  function test_callWithExactGasAndConsumeAll() external {
+
+    vm.pauseGasMetering();
```

Comment:
We don't have to pause/unpause gas monitoring unless we care deeply about this test for the gas snapshot, which we don't here